nuget.config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="AvaloniCI" value="https://www.myget.org/F/avalonia-ci/api/v2" />
+	<add key="AvaloniPRs" value="https://www.myget.org/F/avalonia-prs/api/v2" />
```

Comment:
What's this?

Reply:
see convo below, its not an added dependency. can be removed now that this is merged.